internal/copyUtil/limitCopy.go

```diff
@@ -0,0 +1,38 @@
+package copyUtil
+
+import (
+	"errors"
+	"io"
+)
+
+const (
+	chunkSize = 4 * 1024 * 1024 // 4MB per chunk
+	maxChunks = 100             // Up to 400MB
```

Comment:
I think for the purpose of go-slug where it's gzips, we could reduce the sizing.
```suggestion
	chunkSize = 1* 1024 * 1024 // 1MB per chunk
	maxChunks = 50             // Up to 50MB
```

Reply:
Updated as suggested.